### PR TITLE
Feature/change route plant location

### PR DESCRIPTION
**Pull request by sunilsabatp**
to do -> add route for project site and plantLocation(ploc)

steps-> 1.visit to any project` /:projectID` 
              2.add query params for projectSite(site) and plantLocation (ploc) i.e ->/:projectID?site="sitename"&ploc="HID"
              3.example` /yucatan?site=Las%20Américas%2010&ploc=QYVMCI`

### CODE REVIEW

**Comment by prachigarg19** [comment]
@sunilsabatp Could you look into this? 
image

File: src/features/common/types/project.d.ts [hunk 1]
```diff
@@ -18,3 +18,247 @@ export interface SingleProject {
...
+  timezone_type: number
+}
+
+
+//single plant location 
+
+export interface SinglePlantLocation {
+  metadata: Metadata
+  hid: string
+  otherSpecies: any
```
Comment (prachigarg19) [inline src/features/common/types/project.d.ts:53]: Could you replace any to specific data types?

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/\[p\]**](https://planet-webapp-poywa6lad-planetapp.vercel.app/[p]) |
|:-:|
|<a href="https://planet-webapp-poywa6lad-planetapp.vercel.app/[p]">Screenshot of /[p]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/hFQ-M3brTFDW-BO64exTLBT9yqzj-M_d01vuJzAkKinR-jCz2BFoLseaBPt_NSTrZWeXV8v_U8nOmMEMYJcjSA.png">(view full size)</a> |







Commit 3437da248a34ef93afd0226097ac4ed21b8b0c2d (https://planet-webapp-poywa6lad-planetapp.vercel.app).

File: src/features/common/types/project.d.ts (hunk 1 above)
Comment (sunilsabatp) [reply src/features/common/types/project.d.ts:53]: > @sunilsabatp Could you look into this? image

this issue have been fixed now

**Comment by mariahosfeld** [comment]
Functionality works for me.

**Comment by mariahosfeld** [comment]
After latest updates the functionality still works for me.

**Comment by mariahosfeld** [comment]
Functionality works after fix, redirected to default site when ploc is invalid.

**Review by prachigarg19 — APPROVED** [review]

**Pull request merged**